Internal Server Error: /myadmin-encoded-target/staffs/salary-logs/
Traceback (most recent call last):
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\views\generic\base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\views\generic\base.py", line 143, in dispatch
    return handler(request, *args, **kwargs)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\views\generic\list.py", line 154, in get
    self.object_list = self.get_queryset()
  File "C:\Users\hp\Desktop\fiberswift\djangoprojects\stylezbydee\myadmin\staff.py", line 132, in get_queryset
    SalaryUtils.create_salary_logs()
  File "C:\Users\hp\Desktop\fiberswift\djangoprojects\stylezbydee\myadmin\utils.py", line 98, in create_salary_logs
    date_due = timezone.datetime(year, next_month, 25).date()
UnboundLocalError: local variable 'year' referenced before assignment
"GET /myadmin-encoded-target/staffs/salary-logs/ HTTP/1.1" 500 93897
Internal Server Error: /myadmin-encoded-target/staffs/add/
Traceback (most recent call last):
  File "C:\Users\hp\Desktop\fiberswift\djangoprojects\stylezbydee\myadmin\forms.py", line 135, in save
    staff.user = User.objects.get(account_id=user_id)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\db\models\manager.py", line 87, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\db\models\query.py", line 637, in get
    raise self.model.DoesNotExist(
users.models.User.DoesNotExist: User matching query does not exist.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\views\generic\base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\views\generic\base.py", line 143, in dispatch
    return handler(request, *args, **kwargs)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\views\generic\edit.py", line 184, in post
    return super().post(request, *args, **kwargs)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\views\generic\edit.py", line 153, in post
    return self.form_valid(form)
  File "C:\Users\hp\Desktop\fiberswift\djangoprojects\stylezbydee\myadmin\staff.py", line 40, in form_valid
    return super().form_valid(form)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\views\generic\edit.py", line 135, in form_valid
    self.object = form.save()
  File "C:\Users\hp\Desktop\fiberswift\djangoprojects\stylezbydee\myadmin\forms.py", line 137, in save
    raise forms.ValidationError(
django.core.exceptions.ValidationError: ['No user found with User ID: 34534563545']
"POST /myadmin-encoded-target/staffs/add/ HTTP/1.1" 500 118294
Internal Server Error: /myadmin-encoded-target/staffs/1/
Traceback (most recent call last):
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\response.py", line 90, in rendered_content
    template = self.resolve_template(self.template_name)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\response.py", line 72, in resolve_template
    return select_template(template, using=self.using)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\loader.py", line 42, in select_template
    return engine.get_template(template_name)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\backends\django.py", line 33, in get_template
    return Template(self.engine.get_template(template_name), self)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\engine.py", line 175, in get_template
    template, origin = self.find_template(template_name)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\engine.py", line 157, in find_template
    template = loader.get_template(name, skip=skip)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\base.py", line 200, in compile_nodelist
    return parser.parse()
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\base.py", line 513, in parse
    raise self.error(token, e)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\base.py", line 511, in parse
    compiled_result = compile_func(self, token)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\loader_tags.py", line 293, in do_extends
    nodelist = parser.parse()
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\base.py", line 513, in parse
    raise self.error(token, e)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\base.py", line 511, in parse
    compiled_result = compile_func(self, token)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\loader_tags.py", line 232, in do_block
    nodelist = parser.parse(("endblock",))
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\base.py", line 513, in parse
    raise self.error(token, e)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\base.py", line 511, in parse
    compiled_result = compile_func(self, token)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\defaulttags.py", line 953, in do_if
    nodelist = parser.parse(("elif", "else", "endif"))
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\base.py", line 513, in parse
    raise self.error(token, e)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\base.py", line 511, in parse
    compiled_result = compile_func(self, token)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\defaulttags.py", line 853, in do_for
    nodelist_loop = parser.parse(
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\base.py", line 484, in parse
    raise self.error(token, e)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\base.py", line 482, in parse
    filter_expression = self.compile_filter(token.contents)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\base.py", line 600, in compile_filter
    return FilterExpression(token, self)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\base.py", line 698, in __init__
    filter_func = parser.find_filter(filter_name)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\base.py", line 606, in find_filter
    raise TemplateSyntaxError("Invalid filter: '%s'" % filter_name)
django.template.exceptions.TemplateSyntaxError: Invalid filter: 'intcomma'
"GET /myadmin-encoded-target/staffs/1/ HTTP/1.1" 500 269661
Internal Server Error: /myadmin-encoded-target/staffs/4/
Traceback (most recent call last):
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\db\backends\utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\db\backends\sqlite3\base.py", line 328, in execute
    return super().execute(query, params)
sqlite3.OperationalError: no such column: myadmin_staffsalarylog.remarks

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\response.py", line 92, in rendered_content
    return template.render(context, self._request)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\base.py", line 175, in render
    return self._render(context)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\base.py", line 167, in _render
    return self.nodelist.render(context)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\loader_tags.py", line 157, in render
    return compiled_parent._render(context)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\base.py", line 167, in _render
    return self.nodelist.render(context)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\loader_tags.py", line 63, in render
    result = block.nodelist.render(context)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\defaulttags.py", line 320, in render
    if match:
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\db\models\query.py", line 412, in __bool__
    self._fetch_all()
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\db\models\query.py", line 1881, in _fetch_all
    self._result_cache = list(self._iterable_class(self))
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\db\models\query.py", line 91, in __iter__
    results = compiler.execute_sql(
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\db\models\sql\compiler.py", line 1562, in execute_sql
    cursor.execute(sql, params)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\db\backends\utils.py", line 102, in execute
    return super().execute(sql, params)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\db\backends\utils.py", line 67, in execute
    return self._execute_with_wrappers(
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\db\backends\utils.py", line 80, in _execute_with_wrappers
    return executor(sql, params, many, context)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\db\backends\utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\db\utils.py", line 91, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\db\backends\utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\db\backends\sqlite3\base.py", line 328, in execute
    return super().execute(query, params)
django.db.utils.OperationalError: no such column: myadmin_staffsalarylog.remarks
"GET /myadmin-encoded-target/staffs/4/ HTTP/1.1" 500 233906
Internal Server Error: /myadmin-encoded-target/staffs/4/
Traceback (most recent call last):
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\response.py", line 90, in rendered_content
    template = self.resolve_template(self.template_name)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\response.py", line 72, in resolve_template
    return select_template(template, using=self.using)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\loader.py", line 42, in select_template
    return engine.get_template(template_name)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\backends\django.py", line 33, in get_template
    return Template(self.engine.get_template(template_name), self)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\engine.py", line 175, in get_template
    template, origin = self.find_template(template_name)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\engine.py", line 157, in find_template
    template = loader.get_template(name, skip=skip)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\base.py", line 200, in compile_nodelist
    return parser.parse()
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\base.py", line 513, in parse
    raise self.error(token, e)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\base.py", line 511, in parse
    compiled_result = compile_func(self, token)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\loader_tags.py", line 293, in do_extends
    nodelist = parser.parse()
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\base.py", line 513, in parse
    raise self.error(token, e)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\base.py", line 511, in parse
    compiled_result = compile_func(self, token)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\loader_tags.py", line 232, in do_block
    nodelist = parser.parse(("endblock",))
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\base.py", line 513, in parse
    raise self.error(token, e)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\base.py", line 511, in parse
    compiled_result = compile_func(self, token)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\defaulttags.py", line 953, in do_if
    nodelist = parser.parse(("elif", "else", "endif"))
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\base.py", line 513, in parse
    raise self.error(token, e)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\base.py", line 511, in parse
    compiled_result = compile_func(self, token)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\defaulttags.py", line 853, in do_for
    nodelist_loop = parser.parse(
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\base.py", line 484, in parse
    raise self.error(token, e)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\base.py", line 482, in parse
    filter_expression = self.compile_filter(token.contents)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\base.py", line 600, in compile_filter
    return FilterExpression(token, self)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\base.py", line 699, in __init__
    self.args_check(filter_name, filter_func, args)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\base.py", line 761, in args_check
    raise TemplateSyntaxError(
django.template.exceptions.TemplateSyntaxError: default requires 2 arguments, 1 provided
"GET /myadmin-encoded-target/staffs/4/ HTTP/1.1" 500 262150
Internal Server Error: /myadmin-encoded-target/staffs/4/
Traceback (most recent call last):
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\response.py", line 90, in rendered_content
    template = self.resolve_template(self.template_name)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\response.py", line 72, in resolve_template
    return select_template(template, using=self.using)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\loader.py", line 42, in select_template
    return engine.get_template(template_name)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\backends\django.py", line 33, in get_template
    return Template(self.engine.get_template(template_name), self)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\engine.py", line 175, in get_template
    template, origin = self.find_template(template_name)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\engine.py", line 157, in find_template
    template = loader.get_template(name, skip=skip)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\base.py", line 200, in compile_nodelist
    return parser.parse()
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\base.py", line 513, in parse
    raise self.error(token, e)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\base.py", line 511, in parse
    compiled_result = compile_func(self, token)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\loader_tags.py", line 293, in do_extends
    nodelist = parser.parse()
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\base.py", line 513, in parse
    raise self.error(token, e)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\base.py", line 511, in parse
    compiled_result = compile_func(self, token)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\loader_tags.py", line 232, in do_block
    nodelist = parser.parse(("endblock",))
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\base.py", line 513, in parse
    raise self.error(token, e)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\base.py", line 511, in parse
    compiled_result = compile_func(self, token)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\defaulttags.py", line 953, in do_if
    nodelist = parser.parse(("elif", "else", "endif"))
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\base.py", line 513, in parse
    raise self.error(token, e)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\base.py", line 511, in parse
    compiled_result = compile_func(self, token)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\defaulttags.py", line 853, in do_for
    nodelist_loop = parser.parse(
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\base.py", line 484, in parse
    raise self.error(token, e)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\base.py", line 482, in parse
    filter_expression = self.compile_filter(token.contents)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\base.py", line 600, in compile_filter
    return FilterExpression(token, self)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\base.py", line 699, in __init__
    self.args_check(filter_name, filter_func, args)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\base.py", line 761, in args_check
    raise TemplateSyntaxError(
django.template.exceptions.TemplateSyntaxError: default requires 2 arguments, 1 provided
"GET /myadmin-encoded-target/staffs/4/ HTTP/1.1" 500 262150
Internal Server Error: /account/staffs/profile/
Traceback (most recent call last):
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\response.py", line 90, in rendered_content
    template = self.resolve_template(self.template_name)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\response.py", line 72, in resolve_template
    return select_template(template, using=self.using)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\loader.py", line 42, in select_template
    return engine.get_template(template_name)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\backends\django.py", line 33, in get_template
    return Template(self.engine.get_template(template_name), self)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\engine.py", line 175, in get_template
    template, origin = self.find_template(template_name)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\engine.py", line 157, in find_template
    template = loader.get_template(name, skip=skip)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\base.py", line 200, in compile_nodelist
    return parser.parse()
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\base.py", line 513, in parse
    raise self.error(token, e)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\base.py", line 511, in parse
    compiled_result = compile_func(self, token)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\loader_tags.py", line 293, in do_extends
    nodelist = parser.parse()
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\base.py", line 513, in parse
    raise self.error(token, e)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\base.py", line 511, in parse
    compiled_result = compile_func(self, token)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\loader_tags.py", line 232, in do_block
    nodelist = parser.parse(("endblock",))
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\base.py", line 513, in parse
    raise self.error(token, e)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\base.py", line 511, in parse
    compiled_result = compile_func(self, token)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\defaulttags.py", line 953, in do_if
    nodelist = parser.parse(("elif", "else", "endif"))
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\base.py", line 513, in parse
    raise self.error(token, e)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\base.py", line 511, in parse
    compiled_result = compile_func(self, token)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\defaulttags.py", line 853, in do_for
    nodelist_loop = parser.parse(
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\base.py", line 484, in parse
    raise self.error(token, e)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\base.py", line 482, in parse
    filter_expression = self.compile_filter(token.contents)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\base.py", line 600, in compile_filter
    return FilterExpression(token, self)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\base.py", line 698, in __init__
    filter_func = parser.find_filter(filter_name)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\base.py", line 606, in find_filter
    raise TemplateSyntaxError("Invalid filter: '%s'" % filter_name)
django.template.exceptions.TemplateSyntaxError: Invalid filter: 'intcomma'
"GET /account/staffs/profile/ HTTP/1.1" 500 271214
Internal Server Error: /account/staffs/profile/
Traceback (most recent call last):
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\defaulttags.py", line 1026, in find_library
    return parser.libraries[name]
KeyError: 'intcomma'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\response.py", line 90, in rendered_content
    template = self.resolve_template(self.template_name)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\response.py", line 72, in resolve_template
    return select_template(template, using=self.using)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\loader.py", line 42, in select_template
    return engine.get_template(template_name)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\backends\django.py", line 33, in get_template
    return Template(self.engine.get_template(template_name), self)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\engine.py", line 175, in get_template
    template, origin = self.find_template(template_name)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\engine.py", line 157, in find_template
    template = loader.get_template(name, skip=skip)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\base.py", line 200, in compile_nodelist
    return parser.parse()
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\base.py", line 513, in parse
    raise self.error(token, e)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\base.py", line 511, in parse
    compiled_result = compile_func(self, token)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\loader_tags.py", line 293, in do_extends
    nodelist = parser.parse()
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\base.py", line 513, in parse
    raise self.error(token, e)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\base.py", line 511, in parse
    compiled_result = compile_func(self, token)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\defaulttags.py", line 1088, in load
    lib = find_library(parser, name)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\defaulttags.py", line 1028, in find_library
    raise TemplateSyntaxError(
django.template.exceptions.TemplateSyntaxError: 'intcomma' is not a registered tag library. Must be one of:
admin_list
admin_modify
admin_tree
admin_tree_list
admin_urls
cache
cloudinary
cloudinary_static
core
crispy_forms_field
crispy_forms_filters
crispy_forms_tags
crispy_forms_utils
humanize
i18n
l10n
log
myadmin
rest_framework
shop
static
tz
unfold
unfold_list
vocabulary
"GET /account/staffs/profile/ HTTP/1.1" 500 239802
Internal Server Error: /account/staffs/profile/
Traceback (most recent call last):
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\defaulttags.py", line 1026, in find_library
    return parser.libraries[name]
KeyError: 'intcomma'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\response.py", line 90, in rendered_content
    template = self.resolve_template(self.template_name)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\response.py", line 72, in resolve_template
    return select_template(template, using=self.using)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\loader.py", line 42, in select_template
    return engine.get_template(template_name)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\backends\django.py", line 33, in get_template
    return Template(self.engine.get_template(template_name), self)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\engine.py", line 175, in get_template
    template, origin = self.find_template(template_name)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\engine.py", line 157, in find_template
    template = loader.get_template(name, skip=skip)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\base.py", line 200, in compile_nodelist
    return parser.parse()
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\base.py", line 513, in parse
    raise self.error(token, e)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\base.py", line 511, in parse
    compiled_result = compile_func(self, token)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\loader_tags.py", line 293, in do_extends
    nodelist = parser.parse()
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\base.py", line 513, in parse
    raise self.error(token, e)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\base.py", line 511, in parse
    compiled_result = compile_func(self, token)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\defaulttags.py", line 1088, in load
    lib = find_library(parser, name)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\defaulttags.py", line 1028, in find_library
    raise TemplateSyntaxError(
django.template.exceptions.TemplateSyntaxError: 'intcomma' is not a registered tag library. Must be one of:
admin_list
admin_modify
admin_tree
admin_tree_list
admin_urls
cache
cloudinary
cloudinary_static
core
crispy_forms_field
crispy_forms_filters
crispy_forms_tags
crispy_forms_utils
humanize
i18n
l10n
log
myadmin
rest_framework
shop
static
tz
unfold
unfold_list
vocabulary
"GET /account/staffs/profile/ HTTP/1.1" 500 239802
Internal Server Error: /account/staffs/profile/
Traceback (most recent call last):
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\defaulttags.py", line 1026, in find_library
    return parser.libraries[name]
KeyError: 'intcomma'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\response.py", line 90, in rendered_content
    template = self.resolve_template(self.template_name)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\response.py", line 72, in resolve_template
    return select_template(template, using=self.using)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\loader.py", line 42, in select_template
    return engine.get_template(template_name)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\backends\django.py", line 33, in get_template
    return Template(self.engine.get_template(template_name), self)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\engine.py", line 175, in get_template
    template, origin = self.find_template(template_name)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\engine.py", line 157, in find_template
    template = loader.get_template(name, skip=skip)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\base.py", line 200, in compile_nodelist
    return parser.parse()
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\base.py", line 513, in parse
    raise self.error(token, e)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\base.py", line 511, in parse
    compiled_result = compile_func(self, token)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\loader_tags.py", line 293, in do_extends
    nodelist = parser.parse()
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\base.py", line 513, in parse
    raise self.error(token, e)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\base.py", line 511, in parse
    compiled_result = compile_func(self, token)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\defaulttags.py", line 1088, in load
    lib = find_library(parser, name)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\defaulttags.py", line 1028, in find_library
    raise TemplateSyntaxError(
django.template.exceptions.TemplateSyntaxError: 'intcomma' is not a registered tag library. Must be one of:
admin_list
admin_modify
admin_tree
admin_tree_list
admin_urls
cache
cloudinary
cloudinary_static
core
crispy_forms_field
crispy_forms_filters
crispy_forms_tags
crispy_forms_utils
humanize
i18n
l10n
log
myadmin
rest_framework
shop
static
tz
unfold
unfold_list
vocabulary
"GET /account/staffs/profile/ HTTP/1.1" 500 239802
Internal Server Error: /account/staffs/profile/
Traceback (most recent call last):
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\defaulttags.py", line 1026, in find_library
    return parser.libraries[name]
KeyError: 'intcomma'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\response.py", line 90, in rendered_content
    template = self.resolve_template(self.template_name)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\response.py", line 72, in resolve_template
    return select_template(template, using=self.using)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\loader.py", line 42, in select_template
    return engine.get_template(template_name)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\backends\django.py", line 33, in get_template
    return Template(self.engine.get_template(template_name), self)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\engine.py", line 175, in get_template
    template, origin = self.find_template(template_name)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\engine.py", line 157, in find_template
    template = loader.get_template(name, skip=skip)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\base.py", line 200, in compile_nodelist
    return parser.parse()
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\base.py", line 513, in parse
    raise self.error(token, e)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\base.py", line 511, in parse
    compiled_result = compile_func(self, token)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\loader_tags.py", line 293, in do_extends
    nodelist = parser.parse()
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\base.py", line 513, in parse
    raise self.error(token, e)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\base.py", line 511, in parse
    compiled_result = compile_func(self, token)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\defaulttags.py", line 1088, in load
    lib = find_library(parser, name)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\defaulttags.py", line 1028, in find_library
    raise TemplateSyntaxError(
django.template.exceptions.TemplateSyntaxError: 'intcomma' is not a registered tag library. Must be one of:
admin_list
admin_modify
admin_tree
admin_tree_list
admin_urls
cache
cloudinary
cloudinary_static
core
crispy_forms_field
crispy_forms_filters
crispy_forms_tags
crispy_forms_utils
humanize
i18n
l10n
log
myadmin
rest_framework
shop
static
tz
unfold
unfold_list
vocabulary
"GET /account/staffs/profile/ HTTP/1.1" 500 239802
Internal Server Error: /account/staffs/profile/update/
Traceback (most recent call last):
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\views\generic\base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\contrib\auth\mixins.py", line 73, in dispatch
    return super().dispatch(request, *args, **kwargs)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\views\generic\base.py", line 143, in dispatch
    return handler(request, *args, **kwargs)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\views\generic\edit.py", line 203, in get
    self.object = self.get_object()
  File "C:\Users\hp\Desktop\fiberswift\djangoprojects\stylezbydee\users\staff\profile.py", line 29, in get_object
    return get_object_or_404(self.model,user = self.request.user)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\shortcuts.py", line 85, in get_object_or_404
    return queryset.get(*args, **kwargs)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\db\models\query.py", line 623, in get
    clone = self._chain() if self.query.combinator else self.filter(*args, **kwargs)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\db\models\query.py", line 1436, in filter
    return self._filter_or_exclude(False, args, kwargs)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\db\models\query.py", line 1454, in _filter_or_exclude
    clone._filter_or_exclude_inplace(negate, args, kwargs)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\db\models\query.py", line 1461, in _filter_or_exclude_inplace
    self._query.add_q(Q(*args, **kwargs))
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\db\models\sql\query.py", line 1546, in add_q
    clause, _ = self._add_q(q_object, self.used_aliases)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\db\models\sql\query.py", line 1577, in _add_q
    child_clause, needed_inner = self.build_filter(
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\db\models\sql\query.py", line 1427, in build_filter
    lookups, parts, reffed_expression = self.solve_lookup_type(arg, summarize)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\db\models\sql\query.py", line 1237, in solve_lookup_type
    _, field, _, lookup_parts = self.names_to_path(lookup_splitted, self.get_meta())
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\db\models\sql\query.py", line 1725, in names_to_path
    raise FieldError(
django.core.exceptions.FieldError: Cannot resolve keyword 'user' into field. Choices are: about, access_permission_level, account_id, address, auth_token, carts, city, country, custom_permissions, date_joined, date_of_birth, display_picture, email, email_verified, full_name, gender, groups, id, is_active, is_staff, is_superuser, last_login, logentry, notification, ordernote, orders, password, password_reset_tokens, phone_number, phone_number_verified, product_reviews, product_saved, security, setting, shipping_detail, staff, user_permissions, user_type
"GET /account/staffs/profile/update/ HTTP/1.1" 500 141596
Internal Server Error: /myadmin-encoded-target/catalogue/create/
Traceback (most recent call last):
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\views\generic\base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\views\generic\base.py", line 143, in dispatch
    return handler(request, *args, **kwargs)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\views\generic\edit.py", line 184, in post
    return super().post(request, *args, **kwargs)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\views\generic\edit.py", line 153, in post
    return self.form_valid(form)
  File "C:\Users\hp\Desktop\fiberswift\djangoprojects\stylezbydee\myadmin\catalogue.py", line 88, in form_valid
    return HttpResponseRedirect(self.success_url)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\http\response.py", line 616, in __init__
    self["Location"] = iri_to_uri(redirect_to)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\utils\encoding.py", line 137, in iri_to_uri
    iri = str(iri)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\utils\functional.py", line 155, in __text_cast
    return func(*self.__args, **self.__kw)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\urls\base.py", line 88, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\urls\resolvers.py", line 828, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'catalogue-list' not found. 'catalogue-list' is not a valid view function or pattern name.
"POST /myadmin-encoded-target/catalogue/create/ HTTP/1.1" 500 119280
Internal Server Error: /myadmin-encoded-target/catalogue/create/
Traceback (most recent call last):
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\response.py", line 92, in rendered_content
    return template.render(context, self._request)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\base.py", line 175, in render
    return self._render(context)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\base.py", line 167, in _render
    return self.nodelist.render(context)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\loader_tags.py", line 157, in render
    return compiled_parent._render(context)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\base.py", line 167, in _render
    return self.nodelist.render(context)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\template\defaulttags.py", line 471, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\urls\base.py", line 88, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\urls\resolvers.py", line 828, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'calalogue-list' not found. 'calalogue-list' is not a valid view function or pattern name.
"GET /myadmin-encoded-target/catalogue/create/ HTTP/1.1" 500 154993
Internal Server Error: /myadmin-encoded-target/catalogue/create/
Traceback (most recent call last):
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\views\generic\base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\views\generic\base.py", line 143, in dispatch
    return handler(request, *args, **kwargs)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\views\generic\edit.py", line 184, in post
    return super().post(request, *args, **kwargs)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\views\generic\edit.py", line 153, in post
    return self.form_valid(form)
  File "C:\Users\hp\Desktop\fiberswift\djangoprojects\stylezbydee\myadmin\catalogue.py", line 86, in form_valid
    image_formset.save()
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\forms\models.py", line 784, in save
    return self.save_existing_objects(commit) + self.save_new_objects(commit)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\forms\models.py", line 944, in save_new_objects
    self.new_objects.append(self.save_new(form, commit=commit))
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\forms\models.py", line 1142, in save_new
    return super().save_new(form, commit=commit)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\forms\models.py", line 757, in save_new
    return form.save(commit=commit)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\forms\models.py", line 542, in save
    self.instance.save()
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\db\models\base.py", line 814, in save
    self.save_base(
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\db\models\base.py", line 877, in save_base
    updated = self._save_table(
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\db\models\base.py", line 1020, in _save_table
    results = self._do_insert(
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\db\models\base.py", line 1061, in _do_insert
    return manager._insert(
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\db\models\manager.py", line 87, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\db\models\query.py", line 1805, in _insert
    return query.get_compiler(using=using).execute_sql(returning_fields)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\db\models\sql\compiler.py", line 1821, in execute_sql
    for sql, params in self.as_sql():
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\db\models\sql\compiler.py", line 1745, in as_sql
    value_rows = [
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\db\models\sql\compiler.py", line 1746, in <listcomp>
    [
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\db\models\sql\compiler.py", line 1747, in <listcomp>
    self.prepare_value(field, self.pre_save_val(field, obj))
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\db\models\sql\compiler.py", line 1695, in pre_save_val
    return field.pre_save(obj, add=True)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\db\models\fields\files.py", line 317, in pre_save
    file.save(file.name, file.file, save=False)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\db\models\fields\files.py", line 93, in save
    self.name = self.storage.save(name, content, max_length=self.field.max_length)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\core\files\storage\base.py", line 49, in save
    name = self._save(name, content)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\cloudinary_storage\storage.py", line 68, in _save
    response = self._upload(name, content)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\cloudinary_storage\storage.py", line 62, in _upload
    return cloudinary.uploader.upload(content, **options)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\cloudinary\uploader.py", line 45, in upload
    return call_cacheable_api("upload", params, file=file, **options)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\cloudinary\uploader.py", line 462, in call_cacheable_api
    result = call_api(action, params, http_headers, return_error, unsigned, file, timeout, **options)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\cloudinary\uploader.py", line 527, in call_api
    raise Error(result["error"]["message"])
cloudinary.exceptions.Error: Invalid Signature b3c3d14671a4530c01170684df3d7a880e246042. String to sign - 'folder=media/catalogue_images&tags=media&timestamp=1734494803&use_filename=1'.
"POST /myadmin-encoded-target/catalogue/create/ HTTP/1.1" 500 215757
Internal Server Error: /myadmin-encoded-target/catalogue/create/
Traceback (most recent call last):
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\views\generic\base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\views\generic\base.py", line 143, in dispatch
    return handler(request, *args, **kwargs)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\views\generic\edit.py", line 184, in post
    return super().post(request, *args, **kwargs)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\views\generic\edit.py", line 153, in post
    return self.form_valid(form)
  File "C:\Users\hp\Desktop\fiberswift\djangoprojects\stylezbydee\myadmin\catalogue.py", line 86, in form_valid
    image_formset.save()
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\forms\models.py", line 784, in save
    return self.save_existing_objects(commit) + self.save_new_objects(commit)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\forms\models.py", line 944, in save_new_objects
    self.new_objects.append(self.save_new(form, commit=commit))
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\forms\models.py", line 1142, in save_new
    return super().save_new(form, commit=commit)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\forms\models.py", line 757, in save_new
    return form.save(commit=commit)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\forms\models.py", line 542, in save
    self.instance.save()
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\db\models\base.py", line 814, in save
    self.save_base(
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\db\models\base.py", line 877, in save_base
    updated = self._save_table(
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\db\models\base.py", line 1020, in _save_table
    results = self._do_insert(
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\db\models\base.py", line 1061, in _do_insert
    return manager._insert(
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\db\models\manager.py", line 87, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\db\models\query.py", line 1805, in _insert
    return query.get_compiler(using=using).execute_sql(returning_fields)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\db\models\sql\compiler.py", line 1821, in execute_sql
    for sql, params in self.as_sql():
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\db\models\sql\compiler.py", line 1745, in as_sql
    value_rows = [
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\db\models\sql\compiler.py", line 1746, in <listcomp>
    [
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\db\models\sql\compiler.py", line 1747, in <listcomp>
    self.prepare_value(field, self.pre_save_val(field, obj))
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\db\models\sql\compiler.py", line 1695, in pre_save_val
    return field.pre_save(obj, add=True)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\db\models\fields\files.py", line 317, in pre_save
    file.save(file.name, file.file, save=False)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\db\models\fields\files.py", line 93, in save
    self.name = self.storage.save(name, content, max_length=self.field.max_length)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\core\files\storage\base.py", line 49, in save
    name = self._save(name, content)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\cloudinary_storage\storage.py", line 68, in _save
    response = self._upload(name, content)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\cloudinary_storage\storage.py", line 62, in _upload
    return cloudinary.uploader.upload(content, **options)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\cloudinary\uploader.py", line 45, in upload
    return call_cacheable_api("upload", params, file=file, **options)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\cloudinary\uploader.py", line 462, in call_cacheable_api
    result = call_api(action, params, http_headers, return_error, unsigned, file, timeout, **options)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\cloudinary\uploader.py", line 527, in call_api
    raise Error(result["error"]["message"])
cloudinary.exceptions.Error: Stale request - reported time is 2024-12-18 04:10:22 +0000 which is more than 1 hour ago
"POST /myadmin-encoded-target/catalogue/create/ HTTP/1.1" 500 215177
Internal Server Error: /myadmin-encoded-target/clients/1/bespoke-orders/create/
Traceback (most recent call last):
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\db\backends\utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\db\backends\sqlite3\base.py", line 328, in execute
    return super().execute(query, params)
sqlite3.IntegrityError: UNIQUE constraint failed: fashion_bespokeorder.order_id

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\views\generic\base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\views\generic\base.py", line 143, in dispatch
    return handler(request, *args, **kwargs)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\views\generic\edit.py", line 184, in post
    return super().post(request, *args, **kwargs)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\views\generic\edit.py", line 153, in post
    return self.form_valid(form)
  File "C:\Users\hp\Desktop\fiberswift\djangoprojects\stylezbydee\myadmin\bespoke_orders.py", line 151, in form_valid
    self.object.save()
  File "C:\Users\hp\Desktop\fiberswift\djangoprojects\stylezbydee\fashion\models.py", line 410, in save
    super().save(*args, **kwargs)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\db\models\base.py", line 814, in save
    self.save_base(
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\db\models\base.py", line 877, in save_base
    updated = self._save_table(
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\db\models\base.py", line 1020, in _save_table
    results = self._do_insert(
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\db\models\base.py", line 1061, in _do_insert
    return manager._insert(
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\db\models\manager.py", line 87, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\db\models\query.py", line 1805, in _insert
    return query.get_compiler(using=using).execute_sql(returning_fields)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\db\models\sql\compiler.py", line 1822, in execute_sql
    cursor.execute(sql, params)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\db\backends\utils.py", line 102, in execute
    return super().execute(sql, params)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\db\backends\utils.py", line 67, in execute
    return self._execute_with_wrappers(
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\db\backends\utils.py", line 80, in _execute_with_wrappers
    return executor(sql, params, many, context)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\db\backends\utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\db\utils.py", line 91, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\db\backends\utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\db\backends\sqlite3\base.py", line 328, in execute
    return super().execute(query, params)
django.db.utils.IntegrityError: UNIQUE constraint failed: fashion_bespokeorder.order_id
"POST /myadmin-encoded-target/clients/1/bespoke-orders/create/ HTTP/1.1" 500 214803
Internal Server Error: /myadmin-encoded-target/bespoke-orders/8/update/
Traceback (most recent call last):
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\views\generic\base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\views\generic\base.py", line 143, in dispatch
    return handler(request, *args, **kwargs)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\views\generic\edit.py", line 208, in post
    return super().post(request, *args, **kwargs)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\views\generic\edit.py", line 153, in post
    return self.form_valid(form)
  File "C:\Users\hp\Desktop\fiberswift\djangoprojects\stylezbydee\myadmin\bespoke_orders.py", line 215, in form_valid
    self.object = form.save()
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\forms\models.py", line 542, in save
    self.instance.save()
  File "C:\Users\hp\Desktop\fiberswift\djangoprojects\stylezbydee\fashion\models.py", line 417, in save
    super().save(*args, **kwargs)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\db\models\base.py", line 814, in save
    self.save_base(
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\db\models\base.py", line 877, in save_base
    updated = self._save_table(
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\db\models\base.py", line 990, in _save_table
    updated = self._do_update(
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\db\models\base.py", line 1054, in _do_update
    return filtered._update(values) > 0
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\db\models\query.py", line 1231, in _update
    return query.get_compiler(self.db).execute_sql(CURSOR)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\db\models\sql\compiler.py", line 1984, in execute_sql
    cursor = super().execute_sql(result_type)
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\db\models\sql\compiler.py", line 1549, in execute_sql
    sql, params = self.as_sql()
  File "C:\Users\hp\.virtualenvs\stylezbydee-MONumO1N\lib\site-packages\django\db\models\sql\compiler.py", line 1942, in as_sql
    raise TypeError(
TypeError: Tried to update field fashion.BespokeOrder.last_status_log with a model instance, <BespokeOrderStatusLog: 42853720 - measurement acquired - waiting for a advance on payment. on 2024-12-19 17:09>. Use a value compatible with CharField.
"POST /myadmin-encoded-target/bespoke-orders/8/update/ HTTP/1.1" 500 164296
